scc/ootb-supply-chain-testing-scanning.md

```diff
@@ -163,118 +169,53 @@ spec:
-        name: workspace
-        readOnly: false
-    imagePullSecrets:
-    - name: image-secret
-    initContainers:
-    - args:
-      - -c
-      - ./source/untar-gitrepository.sh $REPOSITORY /workspace
-      command:
-      - /bin/bash
-      image: registry.tanzu.vmware.com/supply-chain-security-tools/grype-templates-image@sha256:36d947257ffd5d962d09e61dc9083f5d0db57dbbde5f5d7c7cd91caa323a2c43
-      imagePullPolicy: IfNotPresent
-      name: repo
-      volumeMounts:
-      - mountPath: /workspace
-        name: workspace
-        readOnly: false
-    restartPolicy: Never
-    volumes:
-    - emptyDir: {}
-      name: workspace
-```
-
-- image scanning (`private-image-scan-template`):
+If you're targetting a namespace that doesn't match the one configured in the
+TAP Profiles (i.e, `grype.namespace` is not the same as the one you're writing
+the Workload to) these can be installed in such namespace by making use of the
+`tanzu package install` command as described in [Install Supply Chain Security
+Tools - Scan](install-components.md#install-scst-scan) ():
 
 
-```
-apiVersion: scanning.apps.tanzu.vmware.com/v1beta1
-kind: ScanTemplate
-metadata:
-  name: private-image-scan-template
-spec:
-  template:
-    containers:
-    - args:
-      - -c
-      - ./image/copy-docker-config.sh /secret-data && ./image/scan-image.sh /workspace
-        scan.xml true
-      command:
-      - /bin/bash
-      image: registry.tanzu.vmware.com/supply-chain-security-tools/grype-templates-image@sha256:36d947257ffd5d962d09e61dc9083f5d0db57dbbde5f5d7c7cd91caa323a2c43
-      imagePullPolicy: IfNotPresent
-      name: scanner
-      resources:
-        limits:
-          cpu: 1000m
-        requests:
-          cpu: 250m
-          memory: 128Mi
-      volumeMounts:
-      - mountPath: /.docker
-        name: docker
-        readOnly: false
-      - mountPath: /workspace
-        name: workspace
-        readOnly: false
-      - mountPath: /secret-data
-        name: registry-cred
-        readOnly: true
-    imagePullSecrets:
-    - name: image-secret
-    restartPolicy: Never
-    volumes:
-    - emptyDir: {}
-      name: docker
-    - emptyDir: {}
-      name: workspace
-    - name: registry-cred
-      secret:
-        secretName: image-secret
-```
-
-Although they can be customized, we recommend sticking with the examples
-above.
-
-
-##### Developer Workload
+-  Create a file named `ootb-supply-chain-basic-values.yaml` that specifies the
+   corresponding values to the properties you want to tweak. For example:
+
+    ```
+    grype:
+      namespace: my-dev-namespace
+      targetImagePullSecret: registry-credentials
+    ```
+
+- With the configuration ready, install the templates:
+
+    ```
+    tanzu package install grype-scanner \
+      --package-name grype.scanning.apps.tanzu.vmware.com \
+      --version 1.0.0 \
+      --namespace my-dev-namespace
+    ```
+
+
+**note**: Although the templates can be customized, we recommend sticking with
```

Comment:
Might want to qualify the recommendation? For getting started it makes sense to suggest the OOTB scanner for the OOTB pipeline, but beyond that we don't want to discourage users from bringing their own scanner.